dist/featureFlags.js

```diff
@@ -216,4 +227,36 @@ angular.module('feature-flags').provider('featureFlags', function() {
     }];
 });
 
+angular.module('release-toggle', [])
```

Comment:
Can we give this a different name? release-toggle is just another way of saying feature-flags. Is this the UI for it? Maybe it should be feature-flag-dialog? Or feature-flag-ui?